index.d.ts

```diff
@@ -121,6 +121,16 @@ declare interface TasksResource {
   markAllNotificationsAsRead(taskId: string, body?: Object, options?: ResourceRequestOptions): Promise<undefined>
 }
 
+declare interface UsersResource {
+  me(options?: ResourceRequestOptions): Promise<Object>
```

Comment:
It should be `me(query?: Object, options?: ResourceRequestOptions): Promise<Object>`

This is because we cannot distinguish if you are intending to specifiy query params or options in GET requests. See [here](https://github.com/workast/workast-sdk-js/blob/master/test/workast.spec.js#L698)